releasenotes/notes/apm-send-all-stats-0cecf5b3a62b192f.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: Some less-frequently-used stats about the behavior of the Trace Agent will now be omitted when zero to reduce overhead.
```

```suggestion
   APM: The Trace Agent now omits infrequently used statistics when their values are zero, reducing overhead.
```
Just a small suggestion for clarity